### PR TITLE
Add « ä Ä Ö Ü » in toAscii()

### PR DESCRIPTION
**Pull request by vlakoff**
To complete diareses on vowels.

### CODE REVIEW

**Comment by danielstjules** [comment]
Thanks!

File: src/Stringy.php [hunk 1]
```diff
@@ -427,7 +427,7 @@ public function toAscii()
     {
         $str = $this->str;
         $charsArray = array(
-            'a'    => array('à', 'á', 'â', 'ã', 'ā', 'ą', 'ă', 'å', 'α', 'ά', 'ἀ',
+            'a'    => array('à', 'á', 'â', 'ä', 'ã', 'ā', 'ą', 'ă', 'å', 'α', 'ά', 'ἀ',
```
Comment (GrahamCampbell) [inline src/Stringy.php]: array no longer aligned correctly

Comment (danielstjules) [inline src/Stringy.php]: A bunch of them have messed up alignment. Gonna fix that in another commit.

Comment (GrahamCampbell) [inline src/Stringy.php]: :+1:

Comment (vlakoff) [inline src/Stringy.php]: I know, they already were, and the orderings are a bit inconsistent too. I just didn't touch this to keep the changes the simplest possible.

Comment (danielstjules) [inline src/Stringy.php]: @vlakoff No worries! I would have done the same. Cleaning all that up is for a separate commit. https://github.com/danielstjules/Stringy/commit/91d8da474190185d1f1cdca9b183359209da32b4 I only fixed alignment - haven't touched order.

**Pull request merged**